test/runtests.jl

```diff
@@ -44,6 +44,9 @@ linreg(x::AbstractVecOrMat, y::AbstractVector) = qr!(simplemm(x)) \ y
     @test isa(lm2.pp.chol, CholeskyPivoted)
     @test lm2.pp.chol.piv == [2, 1]
     @test isapprox(coef(lm1), coef(lm2) .* [1., 10.])
+    lm3 = lm(@formula(y~x), (y=1:25, x=repeat(1:5, 5)), contrasts=Dict(:x=>DummyCoding()))
+    lm4 = lm(@formula(y~x), (y=1:25, x=categorical(repeat(1:5, 5))))
+    @test coef(lm3) == coef(lm4)
```

Comment:
Should also test that it's equal to a fixed reference (like we do for `lm1` above).

Reply:
Done